swift-service/src/main/java/com/facebook/swift/service/ThriftServiceProcessor.java

```diff
@@ -149,6 +149,7 @@ public void onSuccess(Boolean result)
                         @Override
                         public void onFailure(Throwable t)
                         {
+                            LOG.error("Failed to process method [" + method.getName() + "] of service [" + method.getServiceName() + "]", t);
```

Comment:
Let me check that. Maybe it will be necessary to put this log deeper in call hierarchy like in ThriftMethodProcessor.